src/main/java/com/account/panel/AccountPanel.java

```diff
@@ -26,18 +29,20 @@ public AccountPanel (ApplicationContext applicationContext) {
         add(accountInput);
         add(addAccountButton);
         add(deleteAccountButton);
-        addAccountButtonListener = new AddAccountButtonListener();
+        addAccountButtonListener = (AddAccountButtonListener)
```

Comment:
Do we need to have it like a class field ?
I don't see any reference to it in other methods